### PR TITLE
add warning in case of non proportional merge label

### LINKED ISSUE #78: Dealing with merge valid labels interpolation issue
As formerly described, we have an issue on the interpolation process from large to smaller epochs. Why? there is often a "rounding effect" on the number of epochs between larger and smaller epochs (fexample: for 10,75s of recording, we have 10 epochs of 1s and 43 epochs of 250ms).
Solution proposed : we crop the end of ALL our recordings so to have a duration which is a multiple of 0.992s (248 samples).
It's brutal, but it works.
What do you think?

### PR DESCRIPTION
**Pull request by lkorczowski**
- [ ] find a way to avoid non-uniform shift of label during merge_labels_list

closes #78

### CODE REVIEW

File: tinnsleep/utils.py [hunk 1]
```diff
@@ -105,6 +106,7 @@ def merge_labels_list(list_valid_labels, n_epoch_final, merge_fun=np.all):
                                       resampling_factor == 1 for j in range(n_epoch_final)]
 
         else:
+            logger.warning("Interpolating non proportional list, expecting to have non-uniform shift across recording")
```
Comment (RobinGuillard) [inline tinnsleep/utils.py:109]: Doesn't work!!
This "else" mention concerns all the cases but the one where we are moving from small epochs to larger epochs. In our case, we don't want to write this warning if we are moving from big epochs to smaller epochs with proportionality, which it does currently!
If you want this warning to work, you have to write:

`        else:
            if not float(n_epoch_final / n_epoch_before).is_integer():
                       logger.warning("Interpolating non proportional list, expecting to have non-uniform shift across recording")`

Comment (lkorczowski) [reply tinnsleep/utils.py:109]: the warning works on my side.
`merge_labels_list([[True, False, True]], 2)
WARNING:root:Interpolating non proportional list, expecting to have non-uniform shift across recording`

Comment (RobinGuillard) [reply tinnsleep/utils.py:109]: Yes but if you try:
`merge_labels_list([[True, True]], 4)`

you shouldn't have a warning, right?
Try it and see if you don't get the warning ;)...

Comment (lkorczowski) [reply tinnsleep/utils.py:109]: @RobinGuillard it is exactly what I'm fixing now (resampling factor is inverted and upsampling is not managed)

**Comment by lkorczowski** [comment]
obsolete, included in #82

**Pull request closed without merging**